Z/ZipFlow/build_tarballs.jl

```diff
@@ -0,0 +1,23 @@
+using BinaryBuilder
+
+name = "ZipFlow"
+version = v"1.0.0"
+
+sources = [GitSource("https://github.com/madler/zipflow.git",
+                     "d4d73304252504bbade9aa6f34332bbb00de6664")]
+
+script = raw"""
+    cd ${WORKSPACE}/srcdir/zipflow/
+    cc zipflow.c -shared -std=c99 -o libzipflow.${dlext} -I${includedir} -L${libdir} -lz
+    mv libzipflow.${dlext} ${libdir}/
```

Comment:
```suggestion
    cc zipflow.c -shared -std=c99 -o ${libdir}/libzipflow.${dlext} -I${includedir} -L${libdir} -lz
```
:slightly_smiling_face: